radio/src/pulses/pulses.h

```diff
@@ -84,6 +84,16 @@ enum ModuleFlag
   #define IS_DSM2_SERIAL_PROTOCOL(protocol)  (0)
 #endif
 
+#if defined(MULTIMODULE)
+  #define IS_MULTIMODULE_PROTOCOL(protocol)  (protocol==PROTO_MULTIMODULE)
+  #if !defined(DSM2)
+     #error You need to enable DSM2 = PPM for MULTIMODULE support
```

Comment:
But since this is Taranis only and the Taranis already has DSM2 uncoditionally enabled. This define is just as an addtional safeguard to let people out what is wrong when mixing compilation options (as it is possible in next).